kiwi/bootloader/install/grub2.py

```diff
@@ -29,7 +29,7 @@
 
 class BootLoaderInstallGrub2(BootLoaderInstallBase):
     """
-        grub2 bootloader installation for x86 bios platform
+        grub2 bootloader installation
```

Comment:
as long as there are 'if some-arch' in the code... but I don't have strong feelings about it